external-resolver/src/main/scala/org/genivi/sota/resolver/Boot.scala

```diff
@@ -99,7 +102,14 @@ object Boot extends BootApp with Directives with BootMigrations
     DeviceDataMigrator(deviceRegistryClient)
   }
 
-  val messageBusListener = system.actorOf(PackageCreatedListener.props(db, system.settings.config))
+  def dbOp[T<: BusMessage](msg: T): Future[_] = {
```

Comment:
Similar here, better name for `dbOp` and let it just take a `msg: PackageCreated`.